### ISSUE TITLE
TypeError: Cannot read property 'request' of undefined

### ISSUE THREAD

**Issue by duongvanba** [sindresorhus/got#1824]
#### Describe the bug
I have this problem when try to connect facebook. This problem happend after about several hours after app restarted. I don't know why
 - Node.js version: v15.5.0
 - OS & version: Ubuntu 18.04

#### Actual behavior
```bash
4|vipbvls  | TypeError: Cannot read property 'request' of undefined
4|vipbvls  |     at /root/code/server/node_modules/got/dist/source/as-promise/index.js:157:48
4|vipbvls  |     at runMicrotasks (<anonymous>)
4|vipbvls  |     at processTicksAndRejections (node:internal/process/task_queues:93:5)
4|vipbvls  |     at async FacebookAndroidAPI.graphql (/root/code/server/build/src/libs/FacebookAndroidAPI.js:31:22)
4|vipbvls  |     at async FacebookAndroidAPI.get_page_feed_stories (/root/code/server/build/src/libs/FacebookAndroidAPI.js:303:22)
4|vipbvls  |     at async VipBvlcScanner.list_livestream (/root/code/server/build/src/modules/vip-bvls/scanner.js:150:26)
4|vipbvls  |     at async VipBvlcScanner.scan (/root/code/server/build/src/modules/vip-bvls/scanner.js:90:32)
4|vipbvls  |     at async VipBvlcScanner.cron (/root/code/server/build/src/modules/vip-bvls/scanner.js:160:9)
4|vipbvls  |     at async /root/code/server/build/src/decorators/CronJob.js:11:13
```
 
 /root/code/server/node_modules/got/dist/source/as-promise/index.js:157:48
 ```js
const shortcut = (responseType) => {
        const newPromise = (async () => {
            // Wait until downloading has ended
            await promise;
            const { options } = globalResponse.request; // <= globalResponse undefined ?? ?? ?? 
            return parse_body_1.default(globalResponse, responseType, options.parseJson, options.encoding);
        })();
        Object.defineProperties(newPromise, Object.getOwnPropertyDescriptors(promise));
        return newPromise;
    };
```
 

 

#### Checklist

- [x ] I have read the documentation.
- [x ] I have tried my code with the latest version of Node.js and Got.

**Comment by szmarczak** [comment]
Looks like #1543 was forgot in the commits somewhere

**Comment by szmarczak** [comment]
Actually it never made to Got 11, but is fixed in Got 12: ff918fb6dedb6d8b23421497ec890d43f45121b7

**Comment by szmarczak** [comment]
Duplicate of #1540
